### ISSUE TITLE
Expected media type for spdx json/xml documents 

### ISSUE THREAD

**Issue by samj1912** [spdx/spdx-spec#577]
Hello SPDX team. We are trying to integrate spdx support in buildpacks. One of the things that we had a question about was the appropriate media type for spdx documents. It looks like the IANA media type for spdx is registered as text/spdx. Looking at the entry document this seems to be specifically for the spdx tag format. What is the expected media type for json/xml spdx documents?

text/spdx+json / text/spdx+xml? Or would all of them be text/spdx? 

Wondering as we were looking at some other sbom formats on the list

CycloneDX seems to have

application/vnd.cyclonedx+json and application/vnd.cyclonedx+xml respectively.
. Swid seems to have application/swid+xml

Any guidance here would be greatly appreciated.

**Comment by samj1912** [comment]
cc: @nishakm maybe you can help?

**Comment by nishakm** [comment]
@rnjudge Applied on behalf of SPDX. We could also apply for something like `application/org.spdx+json`. It's something we can bring up in the next SPDX tech meeting.

**Comment by rnjudge** [comment]
@samj1912 Yes, the intention of the current IANA SPDX type was intended to represent tag-value format with the intention being to re-visit adding other formats in the future so I suppose the future is upon us :) Definitely worth raising at the next meeting. Thanks for surfacing this.

**Comment by rnjudge** [comment]
Hi @samj1912 - just wanted to update you that I will open an application to add SPDX JSON/XML IANA media types this week.

**Comment by rnjudge** [comment]
The `application/spdx+json` media type is officially approved and recorded with IANA: https://www.iana.org/assignments/media-types/application/spdx+json.

After discussing with a few SPDX folks, it was determined that the XML schema needs further review before officially submitting to IANA. I am having conversations around this now and will try to report back with an estimated time frame for when we can expect this.

**Comment by kestewart** [comment]
Rose discussed:   Only missing media type is now XML.     Need to get follow up for @zvr for review.

**Comment by bact** [comment]
Media type information page on IANA also has a field about file extension.
The tag:value SPDX text file has an extension of ".spdx" in IANA.

For JSON, in SPDX 2.3 spec it suggests ".spdx.json"
https://spdx.github.io/spdx-spec/v2.3/conformance/#44-standard-data-format-requirements

For SPDX 3, there's no suggested file extension in the spec
- https://github.com/spdx/spdx-spec/issues/987
